donkeycar/parts/keras.py

```diff
@@ -55,7 +58,19 @@ def set_optimizer(self, optimizer_type, rate, decay):
             self.model.optimizer = keras.optimizers.RMSprop(lr=rate, decay=decay)
         else:
             raise Exception("unknown optimizer type: %s" % optimizer_type)
-    
+
+    def get_input_shape(self):
+        assert self.model is not None, "Need to load model first"
+        return self.model.inputs[0].shape
+
+    def run(self, img_arr, other_arr=None):
```

Comment:
Nit: add documentation for `other_arr`. Its not obvious what it is actually used for.

Reply:
Yep - makes sense, added full doc string.